static/src/stylesheets/module/site-messages/_fiv-banner.scss

```diff
@@ -0,0 +1,183 @@
+.site-message--fiv-banner {
```

Comment:
this is just a copy of the membership banner one, will be edited when the design is done